johnsonandjohnson/Bodiless-JS
# docs(vital-layout): Add API docs and update docs for vital-layout

## Changes

- Update API Docs
- Update Layout Docs
- Slight adjustments in vital-logo, copyright row and helmet tokens

## Test Instructions
* Smoke test on vital-logo (Default token was added)
* Smoke test on  copyrightrow (export was changed)
* Smoke test on vital-helmet (theme domain moved from base to default)
* Verify API doc : http://localhost:8000/___docs/#/Development/API/@bodiless/vital-layout/README
* Verify component doc : http://localhost:8000/___docs/#/VitalDesignSystem/Components/VitalLayout/

BREAKING CHANGE: 

* Rename of token  vitalHelmet.WithDesktopStatickBody -> vitalHelmet.WithDesktopStaticBody
